### ISSUE TITLE
Trying to fix a previous not-so-great installation, can't remove volume

### ISSUE THREAD

**Issue by datner** [DeterminateSystems/nix-installer#615]
I've read similar issues in the repo already, I've followed the instructions in the error and restarted my mac, but still can't unmount the damn volume..

## Error
```
Error: 
   0: Planner error
   1: Error executing action
   2: Action `encrypt_apfs_volume` errored
   3: The keychain lacks a password for the already existing "Nix Store" volume on disk `disk3`, consider removing the volume with `diskutil apfs deleteVolume "Nix Store"` (if you receive error -69888, you may need to run `launchctl bootout system/org.nixos.darwin-store` and `launchctl bootout system/org.nixos.nix-daemon` first)
```

## Metadata
|key|value|
|--|--|
|**version**|0.11.0|
|**os**|macos|
|**arch**|aarch64|

**Comment by Hoverbear** [comment]
Did you try the commands suggested in the error?

**Comment by datner** [comment]
@Hoverbear of course, what do you take me for? 😆 

I don't even remember what I've done, but I managed to solve this using brute force. Absolutely not the right way to do it, but it worked at least

**Comment by datner** [comment]
I am not even sure if the tool itself could have helped me, I basically needed to nuke the nix partition and mix between this installer and the vanilla installer back and forth until it gave way. Like I said, I really brute forced my way through this.

If the tool would be capable, being able to work within limitations would be great help. For example reusing some existing nix partition and just making checks it shows up as expected. If needed with a note that because you didn't do the mounting, then this or that might happen

**Comment by Hoverbear** [comment]
Yeah -- Part of the complication is on encrypted APFS volumes we need to handle things like decrypting the volume with the volume-specific key and that can be really hard to mix with reusing an existing partition if the user has, say, deleted their keys.

**Comment by datner** [comment]
It's really frustrating needing to write a beginner friendly tool for advanced-user operations. I would consider myself to be intermediate at best, so I was stumped hard. Then again, I could not find a solution for this state anywhere, unrelated to nix. I opened this issue because I was so desperate that its been hours and I haven't gained any grounds hahahahhaa

**Comment by Hoverbear** [comment]
Yeah -- We try to catch situations we're aware of and offer a bit of a helping hand -- Still I wish the situation was better (namely, installing on Mac wasn't so complicated)

**Comment by datner** [comment]
Alls well ends well. Though instead of fixing the not-so-great installation, I just replace one not-so-great with another not-so-great installation hahahahhaa

**Comment by Hoverbear** [comment]
Next time you do this, consider doing a backup of data and an internet recovery maybe? 😅

**Comment by datner** [comment]
To backup what exactly? My nix store? I lost nothing of value in the process. One of the many reasons I use nix in the first place hahahahahaha

**Comment by Hoverbear** [comment]
Autocompletions? As in bash completions?
